konfetti/src/main/java/nl/dionsegijn/konfetti/models/ConfettiConfig.kt

```diff
@@ -8,5 +8,6 @@ data class ConfettiConfig(
     var fadeOut: Boolean = false,
     var timeToLive: Long = 2000L,
     var rotate: Boolean = true,
-    var accelerate: Boolean = true
+    var accelerate: Boolean = true,
+    var rotationSpeedMultiplier: Float = 1f
```

Comment:
Instead of adding this to the ConfettiConfig could you maybe look at one of the modules like `VelocityModule`? Using this mechanism allows people to not only set 1 multiplier but also choose between a random multiplier between a min and a max range. 

The reason this is done is that Konfetti shines the most when there's little `random` added to it to make it feel a little more natural.

Reply:
Addressed this by using the `VelocityModule`. Added another parameter too for variance in the multiplier per particle. Other comments addressed too.